_posts/2017/2017-02-02-programming-beyond-the-comfort-zone-javascript-elixir.md

```diff
@@ -0,0 +1,82 @@
+---
+layout: post
+title: "Programming beyond the comfort zone: JavaScript |> Elixir"
+date: "2017-02-02 07:00:00"
+image: blog-header/drink.jpg
+categories: tech-stories
+authors: ["Erik"]
+---
+
+I'm a JavaScript Engineer.
+I write JavaScript.
+I really like writing JavaScript and I feel very comfortable doing so.
```

Comment:
This is just so everything is on a separate line for diffing right? Nothing to do with linebreaks does it?